src/workerd/io/compatibility-date-test.c++

```diff
@@ -179,7 +179,7 @@ KJ_TEST("compatibility flag parsing") {
       CompatibilityDateValidation::FUTURE_FOR_TEST, true, false);
 
   // nodejs_compat implies nodejs_compat_v2 on or after 2024-08-05
```

Comment:
Should the comment be updated.
I try to never repeat values from the code in a comment because those always get out of sync at some point.
Using a constant + comment can help.